model/textparse/promparse_test.go

```diff
@@ -475,8 +496,10 @@ const (
 
 func BenchmarkParse(b *testing.B) {
 	for parserName, parser := range map[string]func([]byte, *labels.SymbolTable) Parser{
-		"prometheus":  NewPromParser,
-		"openmetrics": NewOpenMetricsParser,
+		"prometheus": NewPromParser,
+		"openmetrics": func(b []byte, st *labels.SymbolTable) Parser {
+			return NewOpenMetricsParser(b, st)
+		},
```

Comment:
we cant!
because our `opts ...OpenMetricsOption` now it counts as technically a 3rd parameter here according to the function signature required 2 lines above

Reply:
Ah right, thanks for the explanation